app/src/main/java/com/poohcom1/playalong/ui/components/VideoPlayer.kt

```diff
@@ -15,7 +13,7 @@ import kotlinx.coroutines.delay
 import kotlinx.coroutines.withContext
 
 @Composable
-fun VideoPlayer(url: String, range: LongRange, tempo: Float) {
+fun VideoPlayer(url: String, range: LongRange, tempo: Float, currentPosition: MutableState<Long>) {
```

Comment:
Don't wrap parameter types in `MutableState`. Parameters that are states should already be reactive